cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -100,6 +100,10 @@ ExternalProject_Add(
     INSTALL_DIR
     "@HUNTER_PACKAGE_INSTALL_PREFIX@"
         # not used, just avoid creating Install/<name> empty directory
+    UPDATE_COMMAND
```

Comment:
Build scheme changed, please update `PACKAGE_INTERNAL_DEPS_ID` of all Boost components:
* https://docs.hunter.sh/en/latest/reference/user-modules/hunter_download.html

Reply:
Ok, I didn't know about PACKAGE_INTERNAL_DEPS_ID, I will fix that